createDerby.groovy

```diff
@@ -54,6 +54,7 @@ lmDS = DataSource.register ("Lm", "LIPID MAPS").asDataSource()
 knapsackDS = DataSource.register ("Cks", "KNApSAcK").asDataSource()
 dtxDS = DataSource.register ("Ect", "EPA CompTox").asDataSource()
 // drugbankDS = BioDataSource.DRUGBANK
+//iupharDS = DataSource.register ("Gpl", "IUPHAR").asDataSource() 
```

Comment:
Change IUPHAR to the name used in the `datasources.txt`.

Reply:
Yes I will!